assets/source/setup-guide/app/style.scss

```diff
@@ -240,9 +257,24 @@ $root: ".woocommerce-setup-guide";
 			&__setup-account {
 
 				.connection-info {
+					// Set minimum height to the height of the spinner preloader.
+					min-height: calc(40px);
```

Comment:
❔ 
Not sure why it needs `calc`? It's a leftover after finding the height of spinner is not exposed? 🤔

Reply:
Thanks for catching this, yes that was a leftover. Fixed in https://github.com/woocommerce/pinterest-for-woocommerce/pull/312/commits/839abef87e4803abdc89becd3dc0cda239523974